README.md

```diff
@@ -40,9 +40,23 @@ https://github.com/protocolbuffers/protobuf/releases
 Then, `go get -u` as usual the following packages:
 
 ```sh
+go get -u google.golang.org/grpc
 go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway
 go get -u github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-go get -u github.com/golang/protobuf/protoc-gen-go
+```
+protoc-gen-go has problem to integrate latest protobuf. Typically you can type
+
+```sh
+go get -u github.com/golang/protobu/{proto,protoc-gen-go}
+```
+In case that you have problems in using grpc golang plugin with protobuf, install it from source:
+
+```sh
+GIT_TAG="v1.2.0" # change as needed
+go get -d -u github.com/golang/protobuf/protoc-gen-go
+git -C "$(go env GOPATH)"/src/github.com/golang/protobuf checkout $GIT_TAG
+go install github.com/golang/protobuf/protoc-gen-go
+
```

Comment:
Instead of this, I think we should just use module syntax to download a specific version:
```bash
$ go get github.com/golang/protobuf/protoc-gen-go@1.2.0
```

What do you think?

Reply:
I tried protoc-gen-go@1.2.0 but it failed. In the case of future conflicts, I would like to recommend to install it from source. Many software like glog, gflags are better to be installed from source I think.

Reply:
`go get` already installs from source. Your commands are equivalent to using go get with a specific version (AFAIK).